src/feedback.js

```diff
@@ -321,5 +321,74 @@ export function initFeedbacks() {
 		}
 	}
 
+	if (this.data.apiVersion >= 6) {
+		const UNAVAILABLE_STATUS = 0
+		const CACHING_STATUS = 1
+
+		feedbacks.nextElementCaching = {
+			type: 'boolean',
+			name: 'Next element caching',
+			description: 'Indicates if the upcoming element is caching',
+			defaultStyle: {
+				bgcolor: red,
+			},
+			options: [],
+			callback: ({ options }) => {
+				if (this.data.playoutRunning && this.data.elementsStatuses[this.data.upcomingElementId] === CACHING_STATUS) {
+					return true
+				}
+			},
+		}
+		feedbacks.nextElementUnavailable = {
+			type: 'boolean',
+			name: 'Next element unavailable',
+			description: 'Indicates if the upcoming element is unavailable',
+			defaultStyle: {
+				bgcolor: yellow,
+			},
+			options: [],
+			callback: ({ options }) => {
+				if (this.data.playoutRunning && this.data.elementsStatuses[this.data.upcomingElementId] === UNAVAILABLE_STATUS) {
+					return true
+				}
+			},
+		}
+		feedbacks.templateInsertStatus = {
+			type: 'advanced',
```

Comment:
You can do this with an advanced feedback, but I would suggest to stay with boolean. Users have more flexibility with boolean and they also can be used in triggers.
You can solve this with one feedback resulting to true if `this.data.templateInsertStatus`is same value as option. Then the users have to add the feedback two times with different option value and different colors to visualize al three states. You can offer this from a preset.

Reply:
Thanks for the suggestions, should be correclty applied now